packages/core-app-api/package.json

```diff
@@ -44,6 +44,7 @@
   "dependencies": {
     "@backstage/config": "workspace:^",
     "@backstage/core-plugin-api": "workspace:^",
+    "@backstage/plugin-auth-react": "workspace:^",
```

Comment:
Can we have this dependency?

Reply:
Best if it can be avoided tbh, but it might get a bit tricky. Thinking it's alright to leave for now. In some way we could even reconsider whether this functionality should even live in `auth-react`, could be a separate new package or even `core-plugin-api`. Don't change anything now though 😁